### ISSUE TITLE
Provide Github actions (and other CI tools)

### ISSUE THREAD

**Issue by javydekoning** [majkinetor/au-packages-template#21]
To support other CI's it would be best to move the logic from: https://github.com/majkinetor/au-packages-template/blob/master/.appveyor.yml#L73-L95 to `update_all.ps1` (or another script/function). 

Variables like branch name and commit message MUST be passed in as params, but it will make it much easier to understand the CI example.

I'm using Github actions, [example here](https://github.com/javydekoning/chocolatey-fanatec-packages/blob/master/.github/workflows/ci.yml)

**Comment by majkinetor** [comment]
Yeah, I don't like any programming in CI yaml BS so I generally agree. I was thinking about adding something like it via #18, but currently have no time for it, if you feel like it give it a go, otherwise, I am not sure when I'll be able to do it.
